Spika/UIFlow/MainFlow/Chats/CustomReactione/CustomReactionsViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  CustomReactionsViewController.swift
+//  Spika
+//
+//  Created by Nikola Barbarić on 08.08.2023..
+//
+
+import Foundation
+import UIKit
+import Combine
+
+class CustomReactionsViewController: BaseViewController {
+    private lazy var mainView = CustomReactionsView(emojiSections: EmojiSection.allCases)
+    var viewModel: CustomReactionsViewModel!
+    var actions: [UIAction]?
+    
+    let selectedEmojiPublisher = PassthroughSubject<String, Never>()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView(mainView)
+        setupBindings()
+        viewModel.getEmojis()
+        mainView.collectionView.reloadData()
+        view.backgroundColor = .secondaryBackground
```

Comment:
Isn't there a methos styleViews where this background change should be called?

Reply:
there is on view, but there is not on view controller